L1Trigger/L1TGlobal/interface/ZdcEnergySumTemplate.h

```diff
@@ -0,0 +1,80 @@
+#ifndef L1Trigger_L1TGlobal_ZdcEnergySumTemplate_h
+#define L1Trigger_L1TGlobal_ZdcEnergySumTemplate_h
+
+/**
+ * \class ZdcEnergySumTemplate
+ *
+ *
+ * Description: L1 Global Trigger energy-sum template.
+ *
+ * Implementation:
+ *    <TODO: enter implementation details>
+ *
+ * \author: Vasile Mihai Ghete - HEPHY Vienna
+ *
+ * $Date$
+ * $Revision$
+ *
+ */
+
+// system include files
+#include <string>
+#include <iosfwd>
+
+// user include files
+
+//   base class
+#include "L1Trigger/L1TGlobal/interface/GlobalCondition.h"
+
+// forward declarations
+
+// class declaration
+class ZdcEnergySumTemplate : public GlobalCondition {
+public:
+  // constructor
+  ZdcEnergySumTemplate();
+
+  // constructor
+  ZdcEnergySumTemplate(const std::string&);
+
+  // constructor
+  ZdcEnergySumTemplate(const std::string&, const l1t::GtConditionType&);
+
+  // copy constructor
+  ZdcEnergySumTemplate(const ZdcEnergySumTemplate&);
+
+  // destructor
+  ~ZdcEnergySumTemplate() override;
+
+  // assign operator
+  ZdcEnergySumTemplate& operator=(const ZdcEnergySumTemplate&);
+
+public:
+  /// typedef for a single object template
```

Comment:
there is no typedef here ?

Reply:
Mmm I was checking and I think that's a leftover of previous comments in previous PRs. 
The purpose of that part is define which set of cuts are associated to the template of a specific object. Here we have a single one.